examples/docs/package.json

```diff
@@ -6,7 +6,7 @@
     "build": "next build",
     "clean": "rimraf .next .turbo",
     "dev": "next",
-    "postbuild": "pagefind --site .next/server/app --output-path .next/static/chunks/pagefind",
+    "postbuild": "pagefind --site .next/server/app --output-path './public/_pagefind'",
```

Comment:
```suggestion
    "postbuild": "pagefind --site .next/server/app --output-path public/_pagefind",
```

Reply:
After removing the single quote, should we add a `\\` to escape the `_`? It seems that `turbo run` needs it

Reply:
how did you run turbo? I did `pnpm build:all` and `_pagefind` was appeared in public directory

Reply:
I first deleted the entire `public/_pagefind` directory and then ran `pnpm build:all`

![image](https://github.com/user-attachments/assets/57411b43-b9bc-42d4-bcdf-168ea4fb2e83)

Reply:
It works! Thanks for pointing that out!